draft/security.md

```diff
@@ -0,0 +1,109 @@
+# Security Working Group
```

Comment:
What exactly is the difference between the Security WG and existing security team? The additional "responsibilities" of the WG over the existing team seem fairly small and could potentially be adopted by the existing team.

Similarly, the current Security Team is intentionally invite only and does a fair amount in private. How do you imagine that working with a conventional "working group" model?